scripts/build.js

```diff
@@ -9,6 +9,7 @@
 
 process.env.NODE_ENV = 'production';
 
+var filesize = require('filesize');
```

Comment:
I was under the impression that the assets have already been gzipped, so I just needed to print out their filesizes?